style.css

```diff
@@ -1,495 +1,1146 @@
+  justify-content: flex-end;
 }
+
 .social_icon {
-	display: inline-block;
-	padding: 0 1rem;
+  display: inline-block;
+  padding: 0 1rem;
 }
+
 .nav-center {
-	padding: 0.5rem;
-	width: 100%;
-	margin: 0 auto;
-	background: transparent;
+  padding: 0.5rem;
+  width: 100%;
+  margin: 0 auto;
+  background: transparent;
 }
+
 .nav-header {
-	display: flex;
-	justify-content: space-between;
-	align-items: center;
-	background: transparent;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: transparent;
 }
+
 .nav-toggle {
-	background-color: transparent;
-	border: transparent;
-	font-size: 1.5rem;
-	cursor: pointer;
-	transition: var(--transition);
+  background-color: transparent;
+  border: transparent;
+  font-size: 1.5rem;
+  cursor: pointer;
+  transition: var(--transition);
 }
+
 .nav-toggle:hover {
-	transform: scale(1.2);
+  transform: scale(1.2);
 }
 
 .nav-links {
-	height: 0;
-	overflow: hidden;
-	transition: var(--transition);
+  height: 0;
+  overflow: hidden;
+  text-align: center;
+  background-color: transparent;
+  padding-top: 1.5rem;
 }
+
 .show-links {
-	height: 380px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  height: 100vh;
+  backdrop-filter: blur(6px);
 }
+
 .nav-link {
-	display: block;
-	padding: 1rem;
-	text-transform: capitalize;
-	font-size: 1rem;
-	letter-spacing: var(--spacing);
-	transition: var(--transition);
-	cursor: pointer;
-	font-weight: 600;
-	color: var(--clr-grey-1);
-}
-.nav-link:hover {
-	color: var(--imperial-red);
+  font-family: sans-serif;
+  display: block;
+  padding: 1rem 1.5rem;
+  text-transform: capitalize;
+  font-size: 1.5rem;
+  letter-spacing: var(--spacing);
+  transition: var(--transition);
+  cursor: pointer;
+  font-weight: 1000;
+  color: var(--clr-black);
+}
+
+.nav-link-btn {
+  display: none;
+}
+
+.active {
+  color: var(--imperial-red);
 }
+
+#close-btn {
+  background: transparent;
+  border: none;
+  font-size: 3rem;
+  position: absolute;
+  top: 0;
+  left: 90%;
+  cursor: pointer;
+}
+
 @media screen and (min-width: 768px) {
-	.navbar {
-		background-color: var(--clr-white);
-	}
-	.nav-icons {
-		display: block;
-	}
-	.nav-center {
-		width: 90%;
-		display: flex;
-		justify-content: space-between;
-		padding: 0.5rem 5rem;
-		flex-wrap: wrap;
-	}
-	.nav-toggle {
-		display: none;
-	}
-	.nav-link {
-		display: inline-block;
-		margin-right: 0.7rem;
-	}
-	.nav-links {
-		height: auto;
-		display: flex;
-	}
-	.nav-link:hover {
-		padding: inherit 0;
-	}
+  .navbar {
+    background-color: var(--clr-white);
+  }
+
+  .nav-icons {
+    display: block;
+  }
+
+  .nav-center {
+    width: 90%;
+    display: flex;
+    justify-content: space-between;
+    padding: 0.5rem 5rem;
+    flex-wrap: wrap;
+  }
+
+  .nav-toggle {
+    display: none;
+  }
+
+  .nav-link {
+    font-family: var(--font-family);
+    font-size: 1rem;
+    display: inline-block;
+    margin-right: 0.7rem;
+    padding: 1rem;
+  }
```

Comment:
This behavior is only for you. I checked this site on multiple devices. Works flawlessly.

Reply:
Hey. To make the layout iPad friendly I would have to use one more breakpoint. But since the project demands the use of only one breakpoint, so I had to make do with it. Thanks for the suggestion though.